Assets/Python/RFCUtils.py

```diff
@@ -550,31 +516,31 @@ def convertPlotCulture(self, plot, iPlayer, iPercent, bOwner):
 		
 		if bOwner: plot.setOwner(iPlayer)
 		
-		#if (pCurrent.isCity()):
-		#	city = pCurrent.getPlotCity()
+		#if (plot.isCity()):
+		#	city = plot.getPlotCity()
 		#	iCivCulture = city.getCulture(iCiv)
 		#	iLoopCivCulture = 0
 		#	for iLoopCiv in range(iNumTotalPlayers):
 		#		if (iLoopCiv != iCiv):
-		#			iLoopCivCulture += city.getCulture(iLoopCiv)				
+		#			iLoopCivCulture += city.getCulture(iLoopCiv)
 		#			city.setCulture(iLoopCiv, city.getCulture(iLoopCiv)*(100-iPercent)/100, True)
 		#	city.setCulture(iCiv, iCivCulture + iLoopCivCulture, True)  
 	
 ##		for iLoopCiv in range(iNumTotalPlayers):
 ##			if (iLoopCiv != iCiv):
-##				iLoopCivCulture = pCurrent.getCulture(iLoopCiv)
-##				iCivCulture = pCurrent.getCulture(iCiv)
-##				pCurrent.setCulture(iLoopCiv, iLoopCivCulture*(100-iPercent)/100, True)
-##				pCurrent.setCulture(iCiv, iCivCulture + iLoopCivCulture*iPercent/100, True)
-		#iCivCulture = pCurrent.getCulture(iCiv)
+##				iLoopCivCulture = plot.getCulture(iLoopCiv)

```

Comment:
That is because argument of the function this code is part of named it plot. Everything in the function itself is called plot, but in the commented section it was still called pCurrent. I only synchronized the commented section with the rest of the function.